### PR TITLE
Sanity: Fix argument_spec and documentation

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/cloudformation_stack_set.py
plugins/modules/cloudfront_distribution.py
plugins/modules/cloudfront_invalidation.py
plugins/modules/cloudwatchevent_rule.py
plugins/modules/data_pipeline.py
plugins/modules/dynamodb_table.py
plugins/modules/ec2_asg.py
plugins/modules/ec2_customer_gateway_info.py
plugins/modules/ec2_elb.py
plugins/modules/ec2_elb_info.py
plugins/modules/ec2_instance.py
plugins/modules/ec2_instance_info.py
plugins/modules/ec2_launch_template.py
plugins/modules/ec2_lc.py
plugins/modules/ec2_lc_info.py
plugins/modules/ec2_metric_alarm.py
plugins/modules/ec2_placement_group_info.py
plugins/modules/ec2_transit_gateway_info.py
plugins/modules/ec2_vpc_endpoint.py
plugins/modules/ec2_vpc_endpoint_info.py
plugins/modules/ec2_vpc_igw_info.py
plugins/modules/ec2_vpc_nacl.py
plugins/modules/ec2_vpc_nacl_info.py
plugins/modules/ec2_vpc_nat_gateway_info.py
plugins/modules/ec2_vpc_peering_info.py
plugins/modules/ec2_vpc_route_table.py
plugins/modules/ec2_vpc_vgw_info.py
plugins/modules/ec2_vpc_vpn.py
plugins/modules/ec2_vpc_vpn_info.py
plugins/modules/ecs_attribute.py
plugins/modules/ecs_service.py
plugins/modules/ecs_service_info.py
plugins/modules/ecs_task.py
plugins/modules/ecs_taskdefinition.py
plugins/modules/efs.py
plugins/modules/efs_info.py
plugins/modules/elasticache.py
plugins/modules/elasticache_subnet_group.py
plugins/modules/elb_application_lb.py
plugins/modules/elb_application_lb_info.py
plugins/modules/elb_classic_lb.py
plugins/modules/elb_classic_lb_info.py
plugins/modules/elb_instance.py
plugins/modules/elb_network_lb.py
plugins/modules/elb_target_group.py
plugins/modules/elb_target_group_info.py
plugins/modules/iam.py
plugins/modules/iam_group.py
plugins/modules/iam_role.py
plugins/modules/iam_user.py
plugins/modules/lambda.py
plugins/modules/rds.py
plugins/modules/rds_instance.py
plugins/modules/rds_subnet_group.py
plugins/modules/redshift.py
plugins/modules/redshift_subnet_group.py
plugins/modules/route53.py
plugins/modules/route53_info.py
plugins/modules/s3_bucket_notification.py
plugins/modules/s3_lifecycle.py
plugins/modules/sns_topic.py
tests/sanity/ignore-2.11.txt

### CODE REVIEW

**Comment by Akasurde** [comment]
@tremble Thanks for reviewing. I will amend the commit message when I will retry this again.

File: plugins/modules/ec2_instance.py [hunk 1]
```diff
@@ -180,6 +182,7 @@
       ebs.iops, and ebs.delete_on_termination.
     - For more information about each parameter, see U(https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_BlockDeviceMapping.html).
     type: list
+    elements: str
```
Comment (jillr) [inline plugins/modules/ec2_instance.py:185]: Ah, this one is a list of dicts actually.

**Comment by jillr** [comment]
The ec2_instance tests pass reliably locally now, the CI flakes were just api thresholds and timeouts from how many tests this one ran in parallel.  We have all green now after rerunning a few failures and I'm happy with this one. Thanks again.

File: plugins/modules/ecs_taskdefinition.py [hunk 2]
```diff
@@ -321,11 +321,11 @@ def main():
         family=dict(required=False, type='str'),
         revision=dict(required=False, type='int'),
         force_create=dict(required=False, default=False, type='bool'),
-        containers=dict(required=False, type='list'),
+        containers=dict(required=False, type='list', elements='str'),
```
Comment (Surgo) [inline plugins/modules/ecs_taskdefinition.py:324]: @jillr I think its `dict` instead of `str`

Comment (Surgo) [reply plugins/modules/ecs_taskdefinition.py:324]: Fixed in #238

**Pull request merged**